bslib.brs

```diff
@@ -21,4 +21,16 @@ function bslib_toString(value)
 
     'when all else fails, just return the type
     return "<" + valueType + ">"
+end function
+
+'
+' Simple ternary function. Given a condition, return the
+' consequent if true, and the alternate if false
+'
+function bslib_iff(condition, consequent, alternate)
```

Comment:
I called this `iff` instead of `ternary` just to save a few characters. If you have a large project with a bunch of source code, `ternary` would start to add up. Plus, `iff` is a [known visual basic function](https://docs.microsoft.com/en-us/office/vba/language/reference/user-interface-help/iif-function) so it still makes sense (at least to me). Thoughts?

Reply:
I like.